src/main/kotlin/uk/gov/dluhc/printapi/service/BankHolidaysDataService.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.dluhc.printapi.service
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import uk.gov.dluhc.bankholidaysdataclient.BankHolidayDataClient
+import uk.gov.dluhc.bankholidaysdataclient.BankHolidayDivision
+import java.time.LocalDate
+
+private val logger = KotlinLogging.logger {}
+
+/**
+ * Service class that uses internal dluhc library client which interacts with S3 bank holidays bucket.
+ * Uses v2 version of AWS S3 BankHolidayDataClient.
+ * Note: The service method that calculates bank holidays is cached.
+ */
+@Service
+class BankHolidaysDataService(
```

Comment:
You'd still be able to use `BankHolidaysDataService` in the rest of this code exactly as you already have. All I'm suggesting is that the actual class itself doesnt live here, it lives in `bank-holidays-data-client-library`, and therefore all your imports for it would be `import uk.gov.dluhc.bankholidaysdataclient.BankHolidaysDataService`

Does that make sense?